tests/pre_commit_hook_test.py

```diff
@@ -44,17 +44,53 @@ def test_file_with_secrets(self, mock_log):
     def test_file_no_secrets(self):
         assert_commit_succeeds('test_data/files/file_with_no_secrets.py')
 
-    def test_baseline(self):
+    @pytest.mark.parametrize(
+        ' has_result, use_private_key_scan,, hook_command, commit_result',
```

Comment:
extra comma? maybe `s/commit_result/commit_succeeds`? (also prefixed space, but less fussy)

Reply:
yep, extra comma. The rename sounds good. I will update in a new commit.